doc/releases/changelog-dev.md

```diff
@@ -88,6 +86,38 @@
   >>> circuit()
   tensor([1.+6.123234e-17j, 0.-6.123234e-17j], requires_grad=True)
   ```
+  
+* The `qml.AmplitudeAmplification` operator is introduced, which is a high-level interface for amplitude amplification and its variants.
+  [(#5160)](https://github.com/PennyLaneAI/pennylane/pull/5160)
+
+  ```python
+  @qml.prod
+  def generator(wires):
+    for wire in wires:
+        qml.Hadamard(wires = wire)
+
+  U = generator(wires = range(3))
+  O = qml.FlipSign(2, wires = range(3))
+
+  dev = qml.device("default.qubit")
+
+  @qml.qnode(dev)
+  def circuit():
+
+    generator(wires = range(3))
+    qml.AmplitudeAmplification(U, O, iters = 5, fixed_point=True, work_wire=3)
+
+    return qml.probs(wires = range(3))
+
+  ```
+  
+  ```pycon
+  >>> print(np.round(circuit(),3))
+  [0.009 0.009 0.94  0.009 0.009 0.009 0.009 0.009]
+
+  ```
+
+<h3>Improvements 🛠</h3>
```

Comment:
This was moved when resolving the merge conflict, please move it back

Reply:
I did it on purpose since I realized that Rlefection was a new feature, not an improvement :)

Reply:
Ahh, ok as long as other things which belong under that heading were properly categorized 👍🏼